main/api/src/mill/api/CompileProblemReporter.scala

```diff
@@ -14,6 +14,7 @@ trait CompileProblemReporter {
   def fileVisited(file: os.Path): Unit
   def printSummary(): Unit
   def finish(): Unit
+  def notifyProgress(percentage: Long, total: Long): Unit
```

Comment:
I'm surprised this is binary compatible. I would have expected that we need to provide a default impl.

Reply:
It wasn't. I added it a default impl in the last commit. Please review again.